sdk/ai/azopenai/example_files_operations_test.go

```diff
@@ -0,0 +1,145 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azopenai_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/streaming"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+)
+
+func ExampleClient_AddUploadPart() {
+	azureOpenAIKey := os.Getenv("AOAI_API_KEY")
+	azureOpenAIEndpoint := os.Getenv("AOAI_ENDPOINT")
+
+	if azureOpenAIKey == "" || azureOpenAIEndpoint == "" {
+		fmt.Fprintf(os.Stderr, "Skipping example, environment variables missing\n")
+		return
+	}
+
+	keyCredential := azcore.NewKeyCredential(azureOpenAIKey)
+
+	client, err := azopenai.NewClientWithKeyCredential(azureOpenAIEndpoint, keyCredential, nil)
+	if err != nil {
+		fmt.Println("Error creating client:", err)
+		return
+	}
+
+	// Create an upload request
+	createUploadResp, err := client.CreateUpload(context.Background(), azopenai.CreateUploadRequest{
```

Comment:
I've been inconsistent about this, but we should use `context.TODO()` in sample code, rather than using `context.Background()`, just to make it obvious that a customer should plumb in a context.

Reply:
There are a few spots where this happens, I won't mark them all.